tests/test_for_support/test_for_biomass.py

```diff
@@ -380,6 +581,21 @@ def test_find_direct_metabolites(model, number):
         assert len(biomass.find_direct_metabolites(model, rxn)) is number
 
 
+@pytest.mark.parametrize("model", [
+    pytest.param("direct_met_no_growth",
```

Comment:
You seem to be mixing different things here. Since we use `pytest-raises`, you can simply say:
```python
[
pytest.mark.raises("direct_met_no_growth", exception=ValueError),
...
]
```

Reply:
For this, I was simply following the template in `test_for_consistency.py`, which uses the same basic format to catch errors (as seen in line 740):

~~~
@pytest.mark.parametrize("model", [
    pytest.param("missing_energy_partner",
                 marks=pytest.mark.raises(exception=ZeroDivisionError)),
    pytest.param("infeasible",
                 marks=pytest.mark.raises(exception=Infeasible))
], indirect=["model"])
def test_find_reactions_with_unbounded_flux_default_condition_errors(model):
    """Expect the number of unbounded and blocked metabolites to be correct."""
    consistency.find_reactions_with_unbounded_flux_default_condition(model)
~~~